datafusion/expr/src/utils.rs

```diff
@@ -83,20 +85,16 @@ pub fn grouping_set_to_exprlist(group_expr: &[Expr]) -> Result<Vec<Expr>> {
     }
 }
 
-/// Recursively walk an expression tree, collecting the unique set of column names
+/// Recursively walk an expression tree, collecting the unique set of columns
```

Comment:
here is a pretty good example of reducing the ceremony required do to a walk of an expr by using `inspect_expr_pre` rather than having to define a visitor explicitly